app/views/header-web/index.njk

```diff
@@ -110,3 +257,377 @@
         }
     }) }}
 {% endblock %}
+
+{% block content %}
+  {{ idskIntroBlock({
+    mainTitleLinkLabel: "viac o ministerstve",
+    bottomMenu: [
+      {
+        item:"Eurofondy",
+        href:"#"
+      },
+      {
+        item:"Kontakty",
+        href:"#"
+      },
+      {
+        item:"Informatizácia",
+        href:"#"
+      },
+      {
+        item:"Investície",
+        href:"#"
+      },
+      {
+        item:"O ministerstve",
+        href:"#"
+      }
+    ],
+    searchEnabled:"true",
+    sideMenu: [
+      {
+        item:"O Eurofondoch",
+        href:"#"
+      },
+      {
+        item:"Aktuality",
+        href:"#"
+      },
+      {
+        item:"O Ministerstve",
+        href:"#"
+      },
+      {
+        item:"Aktuálne výzvy",
+        href:"#"
+      },
+      {
+        item:"Dokumenty",
+        href:"#"
+      }]
+  }) }}
+
+  <hr class="idsk-hr-separator govuk-!-margin-bottom-6">
+
+  <h2 class="govuk-heading-l">Aktuality</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {{ idskCard({
+        "type": "hero",
+        "imgSrc": "/public/assets/images/examples/ministerstvo-aktuality-cover.png",
```

Comment:
Toto by bolo vhodné rovno zmeniť za placeholder-img (public/assets/images/image-placeholder.jpg).
Nedávno som všetky tieto obrázky nahradzoval týmito (na develi to už je)